project_report/templates/report_base.html

```diff
@@ -4,72 +4,123 @@
-        h2 { font-size: 2.0em; }
-        h3 { font-size: 1.3em; }
-        .pagebreak { page-break-before: always; }
+        #edgen_logo { width: 70px; margin-left: auto; margin-right: auto; display: inline-block;; }
+
+        h1 { font-size: 1.4em; font-family:Avantgarde; sans-serif; line-height: 70%; padding-top: 80px; }
+        h2 { font-size: 1.7em; line-height: 160%; font-family:Avantgarde, sans-serif; }
+        h3 { font-size: 1.3em; line-height: 160%; font-family:Avantgarde; sans-serif; }
+        h4 { font-size: 2.7em; line-height: 300%; font-family:Avantgarde; sans-serif; }
+        h5 { font-size: 1.1em; line-height: 140%; font-family:Avantgarde; sans-serif; }
+
+        div.charts {
+           display: block;
+           margin-left: auto;
+           margin-top: auto; }
+
+        table {
+            font-family: arial, sans-serif;
+            width: 100%;
+            text-align: left;
+        }
+        td, th {
+            border: none;
+            text-align: left;
+            padding: 4px;
+            font-size: 1.2em;
+        }
+
+        @page {
+        margin-top: 2cm;
+        margin-bottom: 2cm;
+        }
+
+        div.page-footer { margin-bottom: 5cm; }
+        @media screen {
+          div.divFooter {
+           display: none;
+          }
+        }
+        @media print {
+          div.divFooter {
+            position: fixed;
+            bottom: 0%;
+            left: 40%;
+          }
+        }
     </style>
     <meta charset="UTF-8">
-    {% block head %}
-    <title>Summary for project {{ project_name }}</title>
-    {% endblock %}
+
 </head>
+<div class="divFooter"><img id="edgen_logo" src="http://s29.postimg.org/8qp6u1agn/EG_logo_final_version_colourwhite_cropped.png"/></div>
+    <div id="page-header" style="text-align: center;">
+            <h4>Project Summary </h4>
+    </div>
 <body>
-    <div class="container">
-        <div class="page-header">
-            <img id="edgen_logo" src="http://s29.postimg.org/8qp6u1agn/EG_logo_final_version_colourwhite_cropped.png"/>
-            <h1>Summary for project {{ project_name }}</h1>
-        </div>
-        <div id="general_info">
+
+    <div id="general_info" style="padding-top: 150px; page-break-after: always;">
             {% block general_info %}
-            <h2>General project information</h2>
-            {% for title, data in project_info %}
-            <div class="row">
-                <div class="col-xs-3">{{ title }}</div>
-                <div class="col-xs-9 ">{{ data }}</div>
-            </div>
-            {% endfor %}
-            {% endblock %}
-        </div>
+            <table>
+                {% for title, data in project_info %}
+                <tr>
+                    <td>{{ title }}</td>
+                    <td>{{ data }}</td>
+                </tr>
+                {% endfor %}
+            </table>
+        {% endblock %}
+    </div>
+
+    <div style="text-align: justify; font-family: arial, sans-serif;">
```

Comment:
The css here should be moved to the general style section above and given a class name